### PR TITLE
John's first commit

### PR DESCRIPTION
**Pull request by gartee-john-PFG**
update my super dooper pwsh script
@heathermartinPFG

### CODE REVIEW

**Comment by FerraraMarkPFG** [comment]
I see it

**Comment by KingKimPFG** [comment]
Looks good!

**Comment by HooverJoePFG** [comment]
Go Cubs!

File: gartee.txt [hunk 1]
```diff
@@ -1,4 +1,2 @@
-line 1 
-line 2
-added at nicki's request
```
Comment (heathermartinPFG) [inline gartee.txt:3]: Are you sure you want to delete this?

File: gartee.txt [hunk 2]
```diff
@@ -1,4 +1,2 @@
...
-line 3 exciting
\ No newline at end of file
+john line 1
+john line 2
```
Comment (heathermartinPFG) [inline gartee.txt]: What were you trying to accomplish

**Review by heathermartinPFG — CHANGES_REQUESTED** [review]
doesn't matter

File: gartee.txt (hunk 2 above)
Comment (FerraraMarkPFG) [reply gartee.txt]: Guessing Line 2

File: gartee.txt (hunk 1 above)
Comment (gartee-john-PFG) [reply gartee.txt:3]: yes, because mama said so

File: gartee.txt (hunk 2 above)
Comment (gartee-john-PFG) [reply gartee.txt]: trying to configure a server

**Review by heathermartinPFG — APPROVED** [review]
that looks better

**Review by KingKimPFG — APPROVED** [review]
agree with the changes made.

**Review by FerraraMarkPFG — APPROVED** [review]

**Review by SmithJasonPFG — APPROVED** [review]

**Review by HooverJoePFG — APPROVED** [review]
I agree with Nicki

**Pull request merged**